region/crateria/east.json

```diff
@@ -565,7 +565,7 @@
           "sparking": {
             "runways": [
               {
-                "length": 12,
+                "length": 4,
```

Comment:
There seems to be the same thing with water as on the right side. Does it need two runways as well?